### PR TITLE
Inventory system

### PR DESCRIPTION
**Pull request by RezaKusha**

### CODE REVIEW

File: Assets/Animations/Player/Idle.anim
Comment (URAmiRBin) [inline Assets/Animations/Player/Idle.anim:1]: these files should not change in your branch, pull again from main and make sure that animations are not different from the main branch

File: Assets/Joystick Pack/Prefabs.meta
Comment (URAmiRBin) [inline Assets/Joystick Pack/Prefabs.meta:1]: also these

File: Assets/Scripts/Camera/CameraSwitcher.cs
Comment (URAmiRBin) [inline Assets/Scripts/Camera/CameraSwitcher.cs:1]: and these

File: Assets/Scripts/Model/Inventory.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+public class Inventory
+{
+    private Dictionary<string, int> items = new Dictionary<string, int>();
+
+    public void AddItem(ItemData itemData)
+    {
+        string itemName = itemData.itemName;
+        if (items.ContainsKey(itemName))
+        {
+            Debug.Log("ADD " + itemData.name);
```
Comment (URAmiRBin) [inline Assets/Scripts/Model/Inventory.cs]: remove logs

File: Assets/Scripts/Model/Inventory.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            items[itemName]++;
+        }
+        else
+        {
+            items[itemName] = 1;
+        }
```
Comment (URAmiRBin) [inline Assets/Scripts/Model/Inventory.cs]:
```suggestion
            items[itemName] = 1;
```

File: Assets/Scripts/Model/Inventory.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        if (items.ContainsKey(itemName))
+        {
+            if (items[itemName] > 1)
+            {
+                items[itemName]--;
+            }
+            else
+            {
+                items.Remove(itemName);
+            }
```
Comment (URAmiRBin) [inline Assets/Scripts/Model/Inventory.cs:33]:
```suggestion
            if (items[itemName] > 1)
                items[itemName]--;
            else
                items.Remove(itemName);
```

**Review by URAmiRBin — CHANGES_REQUESTED** [review]

**Comment by RezaKusha** [comment]
i closed this one and opened !23 to fix the files issue

**Pull request closed without merging**